torch/custom_class.h

```diff
@@ -74,10 +74,8 @@ class class_ {
                                                // torch::init<...>()
     auto func = [](c10::tagged_capsule<CurClass> self, Types... args) {
       auto classObj = c10::make_intrusive<CurClass>(args...);
-      auto genericPtr = c10::static_intrusive_pointer_cast<torch::CustomClassHolder>(std::move(classObj));
-      auto capsule = c10::IValue(std::move(genericPtr));
-      auto object = std::move(self.ivalue).toObject();
-      object->setSlot(0, std::move(capsule));
+      auto object = self.ivalue.toObject();
```

Comment:
nit: std::move was dropped here.

Reply:
I don't think `std::move()` was correct here, it was added as a codemod and was moving this IValue out, when the `__init__` method should just modify it in place